pkg/kubectl/cmd/delete.go

```diff
@@ -116,6 +116,7 @@ func RunDelete(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		SelectAllParam(deleteAll).
 		ResourceTypeOrNameArgs(false, args...).RequireObject(false).
 		Flatten().
+		Latest().
```

Comment:
Thanks for the pionter. In my case, I actually don't need to read the objects, I just need to know if the objects exist. Is there a way to do this?